README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+# hw01
+
+# Biography
+### Heather Weller
+My name is **Heather Weller** and I am a 3rd year Public Policy Major and Human Rights Minor. Here are a few facts about me:
+1. I love [narwhals](https://en.wikipedia.org/wiki/Narwhal)
+2. I have started two non-profits named:
+   * Thank a Veteran
+   * Open Access
+3. I am originally from a very small town in Minnesota called New York Mills
+4. I will be doing the 5-year BA/MA Program at Harris next year!
+5. I am in UC Cheer at UChicago (along with many other RSO's)
+
+That's all for now, except for a picture of me that you will find below!
+
+![Image of Heather Cheering](cheer.png)
```

Comment:
I do not have permission to see your image, possibly because it is stored locally. A potential workaround would be putting the image inside your hw01 repository or creating an issue, dragging and dropping your image there, and then copying and pasting the link into your biography. Hope this helps!

Reply:
Hmmm odd, I did add it to the hw01 repository. I was having problems at first when I pushed to GitHub wehre I couldn't see the image, but then I added it to the repository and it finally let me see my image, so I figured it would be visible for everyone else. I wonder what the exact issue is here then.

Reply:
Strange. It looks like I can see it now. No problems for me. Great job embedding it!